README.md

```diff
@@ -153,18 +153,15 @@ Additional information on how IBC works can be found [here](https://ibc.cosmos.n
    >NOTE: Thinking of chains in the config as "source" and "destination" can be confusing. Be aware that most path are bi-directional.
 
    <br>
-   `rly paths fetch` will check for the relevant `path.json` files for ALL configured chains in your config file.  
-   The path meta-data is queried from the [interchain](https://github.com/cosmos/relayer/tree/main/interchain) directory.
+
+   `rly paths fetch` will check for IBC path meta data from the [chain-registry](https://github.com/cosmos/chain-registry/tree/master/_IBC) and add these paths to your config file.
 
      ```shell
      $ rly paths fetch
      ```
    > **NOTE:** Don't see the path metadata for paths you want to relay on?   
    > Please open a PR to add this metadata to the GitHub repo!
```

Comment:
Maybe this should clarify that the PR needs to go to the chain registry, since we've had a handful of PRs opened against relayer that we've had to redirect.

Reply:
We note that in the note below.

The command itself will also note that in the print out for paths it doesn't find.

Reply:
Mark, I mis-interpreted what you were saying... sorry about that. Not a bad idea at all.